decidim-debates/app/types/decidim/debates/debate_type.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Debates
+    DebateType = GraphQL::ObjectType.define do
+      interfaces [
+        -> { Decidim::Core::CategorizableInterface },
+        -> { Decidim::Comments::CommentableInterface },
+        -> { Decidim::Core::ScopableInterface }
```

Comment:
```suggestion
        -> { Decidim::Core::AuthorableInterface }
```

Scopes don't apply to debates, authors does